documentation/source/users/rmg/modules/generateFluxDiagram.rst

```diff
@@ -7,12 +7,32 @@ Generating Flux Diagrams
 The script, ``generateFluxDiagrams.py``, will create a movie out of a completed RMG model
 that shows interconnected arrows between species that represent fluxes.  
 
-To use this method, you just need a completed RMG run.  THe syntax is as follows::
+To use this method, you just need a Chemkin input file and an RMG species dictionary.
+The syntax is as follows::
 
-	python generateFluxDiagram.py input.py
-	
-where ``input.py`` is the input file for the completed RMG run.  The program will use the automatically
-generated file structure to find the other necessary files to create the movie.
+    python generateFluxDiagram.py [-h] [--java] [--no-dlim] [-s SPECIES] [-f]
+                                  [-n N] [-e N] [-c TOL] [-r TOL] [-t S]
+                                  INPUT CHEMKIN DICTIONARY [CHEMKIN_OUTPUT]
```

Comment:
Should the positional arguments be listed before the optional ones?

Reply:
The way I have it right now, is the same way it would be displayed when showing usage information from the command line, so I think I'll keep it this way.